File: API/Entities/Ticket.cs
namespace API.Entities
{
    public class Ticket
    {
        public int Id { get; set; }


        //Solicitare
        public int SolicitareId { get; set; }
        public Solicitare Solicitare { get; set; }


        //Unitate expeditoare
        public int UnitateId { get; set; }
        public Unitate Unitate { get; set; }


        //Status
        public int StatusId { get; set; }
        public Status Status { get; set; }


        //Serviciu Asignat
        public int ServiciuId { get; set; }
        public Serviciu Serviciu { get; set; }



        public int idUserExpeditor { get; set; }

        public string emailSolicitant { get; set; }

        public string tlfnInteriorSolicitant { get; set; }

        public string tlfMobilSolicitant { get; set; }

        public DateTime dataDeschidereTicket { get; set; } = DateTime.Now;
        public string Descriere { get; set; }

        public int idLucratorAsignat { get; set; }
        public bool Valid { get; set; }
    }
}

File: client/src/app/models/ticket.ts
export interface Ticket {
    id: number
    solicitareId: number
    solicitare: Solicitare
    unitateId: number
    unitate: Unitate
    statusId: number
    status: Status
    serviciuId: number
    serviciu: Serviciu2
    idUserExpeditor: number
    emailSolicitant: string
    tlfnInteriorSolicitant: string
    tlfMobilSolicitant: string
    dataDeschidereTicket: string
    descriere: string
    idLucratorAsignat: number
    valid: boolean
  }
  
  export interface Solicitare {
    id: number
    denumire: string
    serviciuAsignatId: number
    serviciu: Serviciu
    valid: boolean
  }
  
  export interface Serviciu {
    id: number
    denumire: string
    prescurtare: string
    valid: boolean
  }
  
  export interface Unitate {
    id: number
    denumire: string
    prescurtare: string
    valid: boolean
  }
  
  export interface Status {
    id: number
    denumire: string
  }
  
  export interface Serviciu2 {
    id: number
    denumire: string
    prescurtare: string
    valid: boolean
  }
  